Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1172,6 +1172,35 @@ std::string getPrimitiveName(
   }
   return name;
 }
+
+/// Helper used to log only once per unsupported primitive mode.
+struct PrimModeLogHelper {
```

Comment:
We usually avoid abbreviating words for clarity, unless the abbreviation is commonly used across the codebase. https://cesium.com/learn/cesium-unreal/ref-doc/style-guide.html#clarity

I think it's also okay to call it a "Logger" here, but that's more of a nitpick than spelling out "Primitive" 😛 

```suggestion
struct PrimitiveModeLogger {
```